src/Serilog.Enrichers.Sensitive/PathMaskingOperator.cs

```diff
@@ -0,0 +1,52 @@
+using System.Diagnostics.CodeAnalysis;
+using System.Text.RegularExpressions;
+
+namespace Serilog.Enrichers.Sensitive;
+
+/// <summary>
+/// Represents a masking operator for path names.
+/// <remarks>Supports path names for Windows, Linux, and macOS.</remarks>
+/// </summary>
+public class PathMaskingOperator : RegexMaskingOperator
+{
+    private readonly IPathWrapper _pathWrapper;
+    private readonly bool _combineMaskWithPath;
```

Comment:
It took me a while to figure out exactly what this meant. Perhaps changing it to `_keepLastPartOfPath`  or `_doNotMaskLastPartOfPath` would be more descriptive and easier to understand for consumers of this.